src/com/facebook/buck/jvm/groovy/GroovycStep.java

```diff
@@ -91,14 +98,61 @@ public String getDescription(ExecutionContext context) {
   }
 
   private ImmutableList<String> createCommand() {
-    ImmutableList.Builder<String> command = ImmutableList.builder();
-    command.addAll(groovyc.getCommandPrefix(resolver))
+    final ImmutableList.Builder<String> command = ImmutableList.builder();
+
+    command.addAll(groovyc.getCommandPrefix(resolver));
+
+    String classpath = Joiner.on(":").join(transform(declaredClasspathEntries, toStringFunction()));
```

Comment:
Same as above; Windows CI would have caught this for us, and I'm not sure how to write a lint rule that would be context-aware enough to know when `:` is safe to use and when it isn't.